demo-project/.eslintrc.json

```diff
@@ -0,0 +1,61 @@
+{
+    "overrides": [
+        {
+            "files": [
+                "*.ts"
+            ],
+            "parser": "@typescript-eslint/parser",
+            "extends": [
+                "plugin:import/recommended",
+                "plugin:@typescript-eslint/recommended"
+            ],
+            "parserOptions": {
+                "ecmaVersion": 2020,
+                "sourceType": "module"
+            },
+            "rules": {
+                "@typescript-eslint/no-unused-vars": "error"
+            },
+            "settings": {
+                "import/resolver": {
+                    "typescript": {}
+                }
+            }
+        },
+        {
+            "files": [
+                "*.html"
+            ],
+            "plugins": [
+                "@html-eslint"
+            ],
+            "parser": "@html-eslint/parser",
+            "extends": [
+                "plugin:@html-eslint/recommended"
+            ]
+        },
+        {
+            "files": [
+                "*.js"
+            ],
```

Comment:
Shame we have a `js` file. Would it be too much to convert this to a `ts` so we can remove this `js` config part?

Reply:
It's injected in the frontend directly, converting it to TS would need a trsnapile step.

Reply:
Boo that is annoying... Ha